sdk/src/main/java/com/hedera/hashgraph/sdk/ScheduleCreateTransaction.java

```diff
@@ -94,4 +94,11 @@ boolean onFreeze(TransactionBody.Builder bodyBuilder) {
     void onScheduled(SchedulableTransactionBody.Builder scheduled) {
         throw new IllegalStateException("Cannot schedule `ScheduleCreateTransaction`");
     }
+
+    @Override
+    @SuppressWarnings("LiteProtoToString")
+    public String toString() {
```

Comment:
We shouldn't be overriding this method, but instead updating the current `Transaction.toString()` to work even if transaction is not frozen.